cmd/main.go

```diff
@@ -37,6 +42,47 @@ import (
  * --plugin=controller and to start it as agent, we have
  * to pass --plugin=agent.
  */
+
+var (
+	enableLvmMetrics       = flag.Bool("enable-lvm-metrics", true, "If set, LVM metrics will be collected")
+	listenAddress          = flag.String("listen-address", ":9080", "Address on which to expose metrics and web interface.")
+	metricsPath            = flag.String("telemetry-path", "/metrics", "Path under which to expose metrics.")
+	disableExporterMetrics = flag.Bool("disable-exporter-metrics", true, "Exclude metrics about the exporter itself (promhttp_*, process_*, go_*).")
```

Comment:
@iyashu Is `disableExportMetrics` flag required?
If set to false, the exporter will have metrics about itself i.e (process_*, go_*).

Reply:
We have decided to keep it as it might help to debug the exporter.
By default, it will be disabled.

Reply:
Sure, let's keep it disabled by default.